ui/src/app/store/scriptresult/selectors.test.tsx

```diff
@@ -151,6 +151,40 @@ describe("scriptResult selectors", () => {
     ).toStrictEqual([items[0]]);
   });
 
+  it("returns commissioning testing script results by machine id", () => {
```

Comment:
I'd probably rename this `commissioning script results` - having `testing` in there is a bit confusing.

Reply:
Done